tesseract_core/sdk/templates/Dockerfile.base

```diff
@@ -50,8 +64,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 {% endif %}
 
 # Drop to a non-root user
-RUN groupadd -g 1000 tesseractor && \
-    useradd -u 1000 -g 1000 --create-home -s /bin/bash tesseractor
+RUN groupadd -o -g 1000 tesseractor && \
+    useradd -o -u 1000 -g 1000 --create-home -s /bin/bash tesseractor
```

Comment:
What are the practical implications of having non-unique group id/user id? I don't really know this stuff, but [here](https://unix.stackexchange.com/questions/124968/what-are-the-side-effects-of-having-several-unix-users-share-one-uid) they seem to suggest all user entries with same ID are the same user. At the first thought I can't think if there are any drawbacks to this, but you clearly thought about this longer. Any comments?

Reply:
Yes you become the same user, which I think is often even desired – that is, the user we generate will inherit all file ownership and permissions from the existing user 1000.

Can't think of any drawbacks either, so my approach would be to go for it and see if anything breaks later on.

Reply:
great, let's roll with this. i hope we won't build too much downstream dependencies of this subtle behavior